include/openenclave/bits/eeid.h

```diff
@@ -51,6 +51,9 @@ typedef struct _oe_eeid
     /** Entry point of the image. */
     uint64_t entry_point;
 
+    /* Number of pages needed for thread-local data */
+    uint64_t num_tls_pages;
```

Comment:
I think `num_tls_pages` should go into `size_settings`, but there may have been previous discussions about that which I'm not aware of.

Reply:
I think it is inappropriate to put num_tls_page into size_settings, because size_settings is set by the user when building an enclave or creating an enclave, but num_tls_page is not needed.